Lib/site.py

```diff
@@ -268,6 +268,9 @@ def joinuser(*args):
         return joinuser("~", "Library", sys._framework,
                         "%d.%d" % sys.version_info[:2])
 
+    if sys.platform == "vxworks":
```

Comment:
Would you mind to add a short comment to explain that VxWorks OS has no home directories?

Same remark for sysconfig.

Reply:
added for both.